README.md

```diff
@@ -1,12 +1,11 @@
-# crossplane.io
+# Crossplane Documentation
 
-This is the repository for [crossplane.io](https://crossplane.io/) including the [Crossplane documentation](https://crossplane.io/docs).
+This is the repository for the [Crossplane documentation](https://crossplane.io/docs).
 
-Crossplane.io is built using [Hugo](https://gohugo.io/).
+The documentation site is built using [Hugo](https://gohugo.io/) and tested with BrowserStack.
 
 For detailed information about contributing to documentation read the [Docs Contributing Guide](https://crossplane.io/master/contributing/docs/).
```

Comment:
This is stale, but I didn't update because https://github.com/crossplane/docs/pull/260 is already open, and it seems like the link will change again soon with https://github.com/crossplane/docs/pull/271.